admin_scripts/os2displaypc/lts_upgrade_in_place_step_1.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#================================================================
+# HEADER
+#================================================================
+#% SYNOPSIS
+#+    lts_upgrade_in_place_1.sh
+#%
+#% DESCRIPTION
+#%    Step one of the upgrade from 16.04 to 20.04.
+#%
+#================================================================
+#- IMPLEMENTATION
+#-    version         lts_upgrade_in_place_step_1.sh 0.0.1
+#-    author          Carsten Agger
+#-    copyright       Copyright 2020, Magenta Aps
+#-    license         BSD/MIT
+#-    email           info@magenta.dk
+#-
+#================================================================
+#  HISTORY
+#     2021/01/13 : carstena : Script creation
+#
+#================================================================
+# END_OF_HEADER
+#================================================================
+
+# Patch jobmanager to avoid early stoppage.
+sed -i "s/900/900000/" /usr/local/lib/python2.7/dist-packages/bibos_client/jobmanager.py
```

Comment:
Have we ever actually released a Python 2 version of `bibos_client` that contained this logic?

Reply:
To my knowledge we have and  it's running in Copenhagen and Copenhagen only because of their problem with PCs losing connection. Hence the patch.

Reply:
I fixed this to check for the return value & fail if  `sed` fails. I also made sure that if we call this script several times by accident, the logic will only be patched once.